recipes/PolyRound/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "PolyRound" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PolyRound-{{ version }}.tar.gz
+  sha256: 4e8ad50ecd006a21b2df9190e3016474f010ab1bddca328b83a1f7e6d2022213
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.2
+    - pandas >=1.2
+    - python-dateutil >=2.8
+    - python-libsbml >=5.18
+    - scipy >=1.4
+    - optlang >=1.4
+    - cobra >=0.20
+
+test:
+  imports:
+    - PolyRound
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/csb.ethz/PolyRound
+  summary: A python package for rounding polytopes.
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - axel
```

This should be your GitHub username since you're submitting the recipe; this field lists people who are responsible for maintaining the *recipe* on conda-forge. We can ask the upstream maintainer, but it's not at all clear to me that "axel" on GitHub properly corresponds to the upstream on Gitlab.

```suggestion
    - Theorell
```